pkg/operator/options/options.go

```diff
@@ -40,7 +39,10 @@ func init() {
 type nodeIdentitiesValue []string
 
 func newNodeIdentitiesValue(val string, p *[]string) *nodeIdentitiesValue {
-	*p = strings.Split(val, ",")
+	*p = []string{}
+	if val != "" {
+		*p = strings.Split(val, ",")
```

Comment:
The if statement here is a bug fix. Otherwise, it would give an array with 1 element in it for an empty string:
- https://pkg.go.dev/strings#Split